index.html

```diff
@@ -0,0 +1,161 @@
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+              <button>html</button>
+              <button>css</button>
+              <button>javascript</button>
+            </div><br>
+            <button class="btn-proj">See Project</button>
+           </div>
+        </div>
+        
+        <!--ABOUT SECTION-->
+        <div class="about">
+            <h2 class="header">About <br> Myself</h2>
+            <p class="section">
+               I’m a software developer! I can help you build a product,feature or 
+               website Look through some of my work and experience! If you like what 
+               you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+           <p class="footer">LET CONNECT</p>
+             <ul class="imgs">
+              <li><a href="#"><img src="./images/twitter.svg" alt="twitter handle"></a></li>
+              <li><a href="#"><img src="./images/github.jpg" alt="github handle"></a></li>
+              <li><a href="#"><img src="./images/medium.jpg" alt="medium handle"></a></li>
+              <li><a href="#"><img src="./images/angellist.jpg" alt="angellisthandle"></a></li>
+             </ul>
+              <div>
+                <button class="btn-proj">Get my resume</button>
+              </div>
+              <br>
+               <h5 class="lang">Languages</h5>
+                <div class="ellipse">
+                  <button><img src="./images/Ellipse1.png" alt="ellipse"><span>JavaScript</span></button>
+                  <br>
+                  <button><img src="./images/Ellipse2.png" alt="ellipse"><span>HTML</span></button>
+                  <br>
+                   <button><img src="./images/Ellipse3.png" alt="ellipse"><span>CSS</span></button>
+                </div>
+                <div class="parag">
+                  <h4>Frameworks</h4>
+                  <hr>
+                  <p>Skills</p>
+                  <hr>
+                </div>
+                <div class="contact--form">
+                  <form action="https://formspree.io/f/xdoboeky" method="post">
+                    <h2 class="last--para">Contact me</h2>
+                    <p class="section2">If you have an application you are interested in developing,
+                       a feature that you need built or a project that needs coding.
+                        I’d love to help with it 
+                    </p><br>
+                    <input type="text" class="name" name="name" placeholder="Sirri Ryisa" required><br><br>
+                    <input type="email" class="email" name="email" placeholder="sirriryisa@gmail.com" maxlength="30" required><br><br>
+                    <textarea class="msg" name="user_message" placeholder="write your message here" required></textarea><br><br>
```

Comment:
- [ ] First of all, I want to congratulate you for fixing the form 👏  but there are still some requirements missing:
- [ ] The `maxlength="30"` should be in the name field, not the e-mail.
- [ ] The `textarea` should also have a `maxlength` attribute, as you can see here:

![image](https://user-images.githubusercontent.com/66967274/145568808-f1c58b2e-9c77-46ef-ac0f-47746b1aa8a2.png)